### PR TITLE
Auto close other open like footers when opening footer

### PR DESCRIPTION
**Pull request by hakonbo**
(also fixed a couple of typos)
#### APK
[Download build #7458](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7458/artifact/build/artifact/wire-dev-PR97-7458.apk)

### CODE REVIEW

File: app/src/main/java/com/waz/zclient/pages/main/conversation/views/row/message/views/TextMessageWithTimestamp.java [hunk 1]
```diff
@@ -170,21 +170,16 @@ public void setMessage(final Message message) {
             messageTextView.setVisibility(VISIBLE);
         }
         this.message = message;
-        if (messageViewContainer.getTimestampShownSet().contains(message.getId())) {
-            messageViewContainer.setShownTimestampView(this);
-            timestampTextView.setVisibility(VISIBLE);
-        } else {
+//        if (messageViewContainer.getTimestampShownSet().contains(message.getId())) {
```
Comment (shumengye) [inline app/src/main/java/com/waz/zclient/pages/main/conversation/views/row/message/views/TextMessageWithTimestamp.java]: Add explaination for commented out code

File: app/src/main/java/com/waz/zclient/pages/main/conversation/views/row/message/views/TextMessageWithTimestamp.java [hunk 2]
```diff
@@ -170,21 +170,16 @@ public void setMessage(final Message message) {
...
     }
 
     private void expandTimestamp() {
-        if (messageViewContainer.getShownTimestampView() != null &&
-            messageViewContainer.getShownTimestampView() != this) {
-            messageViewContainer.getShownTimestampView().collapseTimestamp();
-        }
-        messageViewContainer.getTimestampShownSet().add(message.getId());
-        messageViewContainer.setShownTimestampView(this);
+//        messageViewContainer.getTimestampShownSet().add(message.getId());
```
Comment (shumengye) [inline app/src/main/java/com/waz/zclient/pages/main/conversation/views/row/message/views/TextMessageWithTimestamp.java]: Add explaination for commented out code

File: app/src/main/java/com/waz/zclient/pages/main/conversation/views/row/message/views/TextMessageWithTimestamp.java [hunk 3]
```diff
@@ -199,7 +194,7 @@ private void expandTimestamp() {
     }
 
     private void collapseTimestamp() {
-        messageViewContainer.getTimestampShownSet().remove(message.getId());
+//        messageViewContainer.getTimestampShownSet().remove(message.getId());
```
Comment (shumengye) [inline app/src/main/java/com/waz/zclient/pages/main/conversation/views/row/message/views/TextMessageWithTimestamp.java]: Add explaination for commented out code

**Comment by hakonbo** [comment]
Just commended out to make it compile. ```TextMessageWithTimestamp``` will be removed once the new timestamp is in place. It's still used for Youtube and a few others, but they will eventually use the new footer. Will be included in the feature branch before merging.

**Pull request merged**